app/controllers/api/v1/visualization/annotations_controller.rb

```diff
@@ -156,6 +155,142 @@ def cell_values
+              key :type, :string
+            end
+            parameter do
+              key :name, :annotations
+              key :in, :query
+              key :description, 'List of annotations'
+              key :example, 'cell_type__ontology_label--group--study,disease__ontology_label'
+              key :required, true
+              key :type, :string
+            end
+            parameter do
+              key :name, :cluster
+              key :in, :query
+              key :description, 'Name of requested cluster'
+              key :type, :string
+              key :required, true
+            end
+            response 200 do
+              key :description, 'Hash of integer-based annotation assignments for all cells in requested cluster'
+              schema do
+                key :title, 'Annotations'
+                property :cells do
+                  key :type, :array
+                  key :description, 'Array of arrays of integer assignments of each cell for all requested annotations'
+                  items do
+                    key :type, :array
+                    key :example, [[0, 0], [0, 1], [1, 2], [2, 2], [2, 0]]
+                    items do
+                      key :type, :integer
+                      key :minItems, 2
+                    end
+                  end
+                end
+                property :facets do
+                  key :type, :array
+                  key :minItems, 2
+                  key :example, [
+                    {
+                      annotation: 'cell_type__ontology_label--group--study',
+                      groups: %w[eosinophil macrophage lymphocyte]
+                    },
+                    {
+                      annotation: 'disease__ontology_label--group--study',
+                      groups: ['Crohn disease', 'acute myeloid leukemia', 'chronic lymphocytic leukemia']
+                    }
+                  ].as_json
+                  items do
+                    key :type, :object
+                    property :annotation do
+                      key :type, :string
+                      key :description, 'Annotation identifier'
+                    end
+                    property :groups do
+                      key :type, :array
+                      key :description, 'List of unique values for requested annotation'
+                      key :example, "['eosinophil', 'macrophage', 'lymphocyte']"
+                      items do
+                        key :type, :string
+                      end
+                    end
+                  end
+                end
+              end
+            end
+            extend SwaggerResponses::StudyControllerResponses
+          end
+        end
+
+        def facets
+          cluster = ClusterVizService.get_cluster_group(@study, params)
+          if cluster.nil?
+            render json: { error: "Cannot find cluster: #{params[:cluster]}" }, status: :not_found and return
```

Comment:
That's a great question, let me check that out.  My gut says "yes, Sentry" but I'm not 100%.

Reply:
As of right now, these are not reported upstream since these are statuses we're explicitly responding with.  An unhandled exception that did generate one of these statuses would be reported to Sentry.